### PR TITLE
Added ajax call to approve/decline button

### PR DESCRIPTION
**Pull request by MibenCoop**
https://github.com/Haufe-Lexware/wicked.haufe.io/issues/203
For some reason, this is a duplicate of this Pull Request 
https://github.com/apim-haufe-io/wicked.ui/pull/46

### CODE REVIEW

**Comment by MibenCoop** [comment]
**Duplicate comment from Martin**: Hey @MibenCoop - we cannot accept a PR if you do not sign the CLA.
@santokhsingh Can you please check and review this PR? Thanks!

**Comment by MibenCoop** [comment]
@santokhsingh  Part of the code is moved to a separate function ( options of the AJAX call ). We can't wrap the whole ajax call, because we need an asynchronous Promise functionality ( .done ).
Sorting and filtering are tested

File: views/admin_approvals.jade [hunk 1]
```diff
@@ -2,24 +2,55 @@ extends layout
...
-                  if (isEmptyGridFilter(filter)) {
-                    d.resolve(data);
+                loadData: function (filter) {
+                    var d = $.Deferred();
+                    $.ajax({
+                      url: "/admin/approvals",
+                      data: filter,
+                      dataType: "json"
+                    }).done(function(result) {
+                      if (!!filter.sortField && !!filter.sortOrder) {
```
Comment (MibenCoop) [inline views/admin_approvals.jade]: Filter object can contain sortField and sortOrder. However, we should applyGridFIlter only to filter fields, not for sorting.

File: views/admin_approvals.jade [hunk 2]
```diff
@@ -2,24 +2,55 @@ extends layout
...
+            type: "POST",
+            dataType: "json",
+            data: {
+              id: item.id,
+              app: item.application.id,
+              api: item.api.id
+            }
+          })
+            .done(function (data, status) {
+                const sorting = $("#approvalsGrid").jsGrid("getSorting");
```
Comment (MibenCoop) [inline views/admin_approvals.jade]: Because of client pagination/sorting/filtering, we should store a state of the grid before reloading

**Comment by santokhsingh** [comment]
@MibenCoop: This is still not correct. You don't need to re-inject filter and sorting. Just grid reloading is suffice to maintain sort and filter state. Please fix it. You might need to fix function which applies filter to ignore properties which are present in filter object and not in item, should be a one liner.

**Pull request closed without merging**